### DISCUSSION TITLE
terminate call if no media is received

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#2583] [General]
Since baresip/re does not support Session Timers (RFC 4028), I started to wonder if it could be possible somehow to detect that media (audio/video) has been not received for a while for a session that is not on hold.

Now if the other party of the call dies or if there is a network failure, baresip does not detect it and the call remains active forever, which is not a good thing to happen.

**Comment by sreimers** [comment]
You can enable [rtp_timeout](https://github.com/baresip/baresip/wiki/Configuration#rtp_timeout)

**Reply by juha-h** [reply]
Thanks
